metaflow/plugins/cards/card_modules/card.py

```diff
@@ -66,7 +67,7 @@ class MetaflowCard(object):
     # FIXME document runtime_data
     runtime_data = None
 
-    def __init__(self, options={}, components=[], graph=None):
```

Comment:
Tweeking function signature here so that default card can access the step function and render it .